inc/Logging/QueryMonitor/RdbBlockBindingOutputHtml.php

```diff
@@ -3,15 +3,15 @@
 namespace RemoteDataBlocks\Logging\QueryMonitor;
 
 use QueryMonitor as QM;
-use QM_Output_Html_Logger;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-if ( class_exists( 'QM_Output_Html_Logger' ) ) {
-	class RdbBlockBindingOutputHtml extends QM_Output_Html_Logger {
+if ( class_exists( __NAMESPACE__ . '\RdbLogOutputHtml' ) ) {
```

Comment:
Do we still need to check for this class existing?

Reply:
Nevermind, I see that `RdbLogOutputHtml` depends on the existence of `QM_Output_Html_Logger`.